src/mixtures/mixturemodel.jl

```diff
@@ -334,7 +334,7 @@ function _mixlogpdf!(r::AbstractArray, d::AbstractMixtureModel, x)
 
             # in the mean time, add log(prior) to lp and
             # update the maximum for each sample
-            for j = 1:n
+            for j in axes(r, 1)
```

Comment:
This should be
```suggestion
            for j in eachindex(r)
```
it seems.